src/graphql/generate.ts

```diff
@@ -0,0 +1,49 @@
+export interface GenerateArgs {
+  groupedTask?: string;
+  singlePrompt?: string;
+}
+
+export interface GenerateParts {
+  singleResult?: string;
+  groupedResult?: string;
+  results: string[];
+}
+
+export class GraphQLGenerate {
+  private groupedTask?: string;
+  private singlePrompt?: string;
+
+  constructor(args: GenerateArgs) {
+    this.groupedTask = args.groupedTask;
+    this.singlePrompt = args.singlePrompt;
+  }
+
+  toString(): string {
+    this.validate();
+
+    let str = 'generate(';
+    const results = ['error'];
+    if (this.singlePrompt) {
+      str += `singleResult:{prompt:"${this.singlePrompt.replace(/[\n\r]+/g, '')}"}`;
```

Comment:
In other clients we have methods that safely quote text when we build the argument, do we have something similar here?

Reply:
Luckily, like the python client, this client is unaffected by this problem